### PR TITLE
link to networkx photo in dec meet minutes was responding 404 , fixed it...

### PR DESCRIPTION
**Pull request by igauravsehrawat**
.... Added a link to contributing.md in readme

### CODE REVIEW

File: content/december-2014-meet-minutes.md [hunk 1]
```diff
@@ -55,11 +55,11 @@ see people forming groups and discussing various topics.
...
 
 After the networking tea break, we had a second round of lightning
-talks. 
+talks.
 
 <a
-href="http://photos1.meetupstatic.com/photos/event/7/0/1/b/highres_432628699.jpeg"><img
-src="http://photos1.meetupstatic.com/photos/event/7/0/1/b/event_432628699.jpeg"
+href="http://photos2.meetupstatic.com/photos/event/7/0/3/a/event_432628730.jpeg"><img
+src="http://photos2.meetupstatic.com/photos/event/7/0/3/a/event_432628730.jpeg"
```
Comment (bravegnu) [inline content/december-2014-meet-minutes.md]: The previous image works for me. Any reason it breaks for you?

File: README.md [hunk 2]
```diff
@@ -49,4 +49,4 @@ running locally. Yay!
 
 ## Contributing
 
-See CONTRIBUTING.md
+See [CONTRIBUTING.md](https://github.com/Chennaipy/website/blob/master/CONTRIBUTING.md)
```
Comment (bravegnu) [inline README.md]: Is it possible to do relative URLs?

File: content/december-2014-meet-minutes.md (hunk 1 above)
Comment (igauravsehrawat) [inline content/december-2014-meet-minutes.md]: [404 for previous image](http://s7.postimg.org/l78tkylx7/universal_404.png)

File: README.md (hunk 2 above)
Comment (igauravsehrawat) [inline README.md]: yes, pushed that commit

File: content/december-2014-meet-minutes.md (hunk 1 above)
Comment (bravegnu) [inline content/december-2014-meet-minutes.md]: OK, I now understand what you are referring to. I am getting the same issue as well. But is this a transient issue? If yes then we might want to rethink our strategy of linking to external resources. Instead we could pull in the external resources once using bower, and serve them from a local copy.

Comment (igauravsehrawat) [inline content/december-2014-meet-minutes.md]: i am not sure about "transient issue",since other links works fine. link was outdated, so fixed it.

Comment (bravegnu) [inline content/december-2014-meet-minutes.md]: OK. I still don't understand why the URL changes over time! But, before we can merge this the `href` should point to the high resolution version of the image. Can you please fix that?

Comment (igauravsehrawat) [inline content/december-2014-meet-minutes.md]: ok. fixed it.

**Pull request merged**